wire.go

```diff
@@ -127,11 +127,13 @@ func (srv *Server) Serve(listener net.Listener) error {
 
 func (srv *Server) serve(ctx context.Context, conn net.Conn) error {
 	ctx = setTypeInfo(ctx, srv.types)
+	ctx = context.WithValue(ctx, "remote_addr", conn.RemoteAddr())
```

Comment:
Would you be able to create a wrapper function to fetch the `remote_addr` from the context? Similar as done within the `conn.go`: https://github.com/jeroenrinzema/psql-wire/blob/main/conn.go#L24-L31